flutter_appauth/ios/Classes/OIDExternalUserAgentIOSNoSSO.m

```diff
@@ -0,0 +1,241 @@
+/*! @file OIDExternalUserAgentIOSNoSSO.m
+    @brief OIDExternalUserAgentIOSNoSSO is a custom user agent based off of the default user agent in the AppAuth iOS SDK found here:
```

Comment:
Minor grammar issue `OIDExternalUserAgentIOSNoSSO is a custom user agent based off of...` here. `OIDExternalUserAgentIOSNoSSO is a custom user agent based on...` should work